### DISCUSSION TITLE
[noob] num layer for the bluetooth/bootloader keys

### DISCUSSION THREAD

**Discussion by enoryw** [urob/zmk-config#69] [Q&A]
Disclaimer: new to ZMK, this is my first config.

So it seems pressing FN layer and smart num (layer) key accesses the symbol layer for the the bluetooth/bootloader keys.

This binding to access the symbol layer also triggers toggling sound on/off--isn't that a bad idea? E.g. sound shouldn't be messed with when bluetooth or flashing firmware is involved.

**Comment by urob** [comment]
This one is indeed a bit tricky :)

The `SYS` layer is activated when both the `FN` and `NUM` layers are simultaneously active. The tricky bit is that the `FN` layer doesn't have a way to activate the `NUM` layer (the corresponding key is overwritten with `C_MUTE`). Hence, one must first activate `NUM` and then `FN`.

TL,DR: Order matters. (1) activate `NUM`, (2) activate `FN`, (3) press `&bootloader`.

I should probably document that somewhere.
